src/common/helper.js

```diff
@@ -1392,6 +1392,34 @@ async function getSkillById (skillId) {
   return _.pick(res.body, ['id', 'name'])
 }
 
+/**
+ * Function to get skills by names
```

Comment:
This description is accurate but not entirely helpful. If I were reading this code, say, to debug an issue, it would be helpful to know that this function:
- calls the standardized skills API to retrieve a list of skills 
- uses exact name matching to find matching skills

Reply:
Fixed